client/contracts/truffle/contracts/oracle/OracleMgr.sol

```diff
@@ -267,4 +276,11 @@ contract OracleMgr is Pausable {
     function addPrice(uint _price) public whenNotPaused onlyOracle onlyValidPrice(_price) {
         price = _price;
     }
+
+    /**
+     * @dev address of a Validator
+     */
+    function getValidatorAddress() onlyOwner public view returns(address){
```

Comment:
I think that this one is no longer necessary.

Reply:
ValidatorMgr variable shouldn't be public? In ValidatorMgr contract, miningTokenAddr is public, shouldn't it be the same case here? If yes, this function is not necessary.
@rgeraldes @yourheropaul ?